### PR TITLE
Only favicon in the <head> tag

### PR DESCRIPTION
**Pull request by douglasprofessional**

### CODE REVIEW

**Comment by TedGoas** [comment]
@douglasprofessional Hi, I'm a little confused why you submitted this. The docs site doesn't currently have a favicon. What would be the benefit to adding all these files?

**Comment by TedGoas** [comment]
@douglasprofessional ping ☝️ just want to make sure you saw my previous comment.

File: docs/index.html [hunk 1]
```diff
@@ -6,6 +6,25 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Cerberus - Patterns for Responsive HTML Email Templates</title>
   <link rel="stylesheet" href="assets/cerberus-docs.css">
+  <!-- favicon -->
+  <link rel="shortcut icon" href="assets/favicon/favicon.ico" type="image/x-icon" />
```
Comment (TedGoas) [inline docs/index.html:10]: @douglasprofessional I think the `ico` might be the only one we want.

Some context:
This repo doesn't get much mobile traffic and I'm not especially concerned with folks bookmarking this on their devices. In the interest of [keeping this simple](http://tedgoas.github.io/Cerberus/#contribute), I vote to start with a single `ico` favicon file to address browsers.

How does that sound? If you agree, would you like to modify this PR? Or would you like me to help?

**Review by TedGoas — CHANGES_REQUESTED** [review]
@douglasprofessional This repo doesn't get much mobile traffic and I'm not especially concerned with folks bookmarking this on their devices. In the interest of [keeping this simple](http://tedgoas.github.io/Cerberus/#contribute), I vote to start with a single `ico` favicon file to address browsers.

How does that sound? If you agree, would you like to modify this PR? Or would you like me to help?

**Comment by douglasprofessional** [comment]
In this case, best to just leave only the line assets/favicon/favicon.ico

You could change, ok?

**Pull request merged**